### DISCUSSION TITLE
Notifications on should be mandatory?

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#133] [Ideas & Features]
A complaint I saw was that a lot of people keep their notifications off (presumably women since they get so many messages) and thus they don't even see the messages. Since our setup will have significantly less spam, and we won't use the "see who likes you, pay to upgrade now" notifications, we could just make it mandatory, so people see it for sure right? I don't know if this is possible tech-wise though.

**Comment by barvian** [comment]
I like this idea for the same reasons, but I think Apple would reject it. From their [review guidelines](https://developer.apple.com/app-store/review/guidelines/#design):

> **4.5.4** Push Notifications must not be required for the app to function

**Reply by WilliamDormer** [reply]
hmm maybe we could make it like "if you don't turn them on, it reduces the effectiveness" but I don't really like doing this frivolously

**Comment by Oshyan** [comment]
I think this is another situation where we should simply *inform and empower* both sides of the user equation. For the person that turns off notifications, a message/warning "You are welcome to disable notifications but please note that this will be publicly shown on your profile so that matches understand you may not see their messages. You may receive fewer matches and some users may choose to filter your profile out of their match queue." And for the other users, like many things we have been discussing, a warning with a popup info box that they can click, saying that this person has notifications disabled, but *also* perhaps a more active one-time notification if they match/message with the person. 

I don't want to sound like a broken record, but I feel similarly about this and a number of other generally good and well-intentioned ideas, so I'm going to outline that a bit more in-depth in another Discussion, it's a little long too, but I do hope you read it: #134

**Comment by ajbott** [comment]
This would restrict user control on an app level, which I don't think would be very good. I would like to see users have the most control as possible. We could compliment other controls that would discourage not having notifications enabled (such as letting other users know how many unread messages they have, how long it takes for them to respond to a message, ect).
